doc/getting_started.md

```diff
@@ -986,17 +1102,17 @@ Add a point and apply point styling using the following code:
   <div class="Carousel-item js-Tabpanes-item--lang js-Tabpanes-item--lang--objective-c">
   {% highlight objc %}
 
-// 1. Create style and position for the Point
+	 // 1. Create style and position for the Point
       NTPointStyleBuilder* pointStyleBuilder = [[NTPointStyleBuilder alloc] init];
 
-      //color is defined as ARGB integer, i.e. following is opaque green
-      //you can not use UIColor (or any other ObjectiveC specific class) in Carto mobile SDK
+      // Color is defined as ARGB integer, i.e. following is opaque green
+      // You can not use UIColor (or any other ObjectiveC specific class) in Carto mobile SDK
      
```

Comment:
CARTO Mobile SDK (CARTO is capitalized, as per our brand guidelines :-)). Lowercase is okay if it is part of code.